### ISSUE TITLE
reexport CPAL

### ISSUE THREAD

**Issue by kamirr** [RustAudio/rodio#274]
I need to use CPAL for audio input but I'd have to take extra care of making sure that my version matches one used by rodio so that my `crate::cpal::SomeStruct` is the same as `rodio::cpal::SomeStruct`. If rodio reexported the entire CPAL module as `rodio::cpal` I wouldn't have have that problem.

Are there any reasons not to do that? If no, I can prepare a pull request.

**Comment by Rua** [comment]
I would like this feature. Do you want to go ahead?

**Comment by est31** [comment]
Makes sense. breaking changes in cpal require breaking changes in rodio already. PRs welcome!
